experimental/federated_plugin/federated_plugin_linux/lib/federated_plugin_linux.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:dbus/dbus.dart';
+import 'package:federated_plugin_platform_interface/federated_plugin_platform_interface.dart';
+import 'package:upower/upower.dart';
+
+class FederatedPluginLinux extends FederatedPluginInterface {
+
+  static void register() {
+    FederatedPluginInterface.instance = FederatedPluginLinux();
+  }
```

Comment:
@domesticmouse @yash1200 Hey folks, just a quick question. What are the steps to register the Dart class as a default instance of Platform interface? Normally for the web we have `registerWith` which is called whenever the web platform is detected to register the plugin class, but what about other platforms? 

Looking at the plugins maintained by the team they use `static void register` to set the default instance, but it doesn't seem to work for me. It throws me unimplementated method error when running on Linux. What am I missing here?

Reply:
Got it, there's currently no way. [Found the issue](https://github.com/flutter/flutter/issues/52267). I'll try to use an alternate option as it's mentioned in the issue. I'll keep track of the issue and keep updating the sample.